ui/app/pages/connected-accounts/connected-accounts.component.js

```diff
@@ -50,7 +51,7 @@ export default class ConnectedAccounts extends PureComponent {
 
     return (
       <Popover
-        title={activeTabOrigin}
+        title={activeTabOrigin === extension.runtime.id ? t('currentExtension') : activeTabOrigin}
```

Comment:
Hm, this will only do it for the MetaMask extension itself, no? I'm just realizing, that's probably a bad idea. We should _probably_ prevent the extension UI from initiating an external permissions request to itself 😅 

That's a different issue. Meanwhile, is there any platform-agnostic way of detecting an extension ID here?

Reply:
hmm good point lol.

Reply:
window.location.protocol seems promising... 'chrome-extension' 'moz-extension' etc